.travis.yml

```diff
@@ -1,20 +1,23 @@
-dist: trusty
+dist: bionic
 language: scala
-jdk:
-- oraclejdk8
+scala:
+  - 2.11.12
+  - 2.12.11
+  - 2.13.2
 addons:
-  postgresql: '9.5'
+  postgresql: '11'
```

Comment:
I think it's advisable to to test against the oldest supported postgres version, which is 9.5 for now.

Reply:
Sure thing, amended that change and left it as 9.5